tests/sentinel/tests/includes/notify.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+leaked_fd_count=`ls /proc/self/fd | grep -vE '0|1|2|3' | wc -l`
```

Comment:
did you mean `ls -l` each fd in a separate line?
what's `3`? do you mean the access to `/proc/<pid>/fd` itself? it won't always be in 3.
seems to me that this would hide a leak of fd `10`, and even `17` and `71`?

Reply:
'^[0|1|2|3]$' will fix the leftover fds, as for the `/proc/<pid>/fd`, any good idea?

Reply:
only way i can think of is to use `ls -l` and match that `*/proc/*/fd` pattern.
so either that (and improve the regex), or we can maybe keep the current code and add some comment.

```
# fd 3 is meant to catch the actual access to /proc/pid/fd, in case there's an fd leak by sentinel,
# it can take 3, but then the access to /proc will take another fd, and we'll catch that
```

Reply:
I would prefer to keep the current code because the `notify.sh` is an extremely simple shell script where the fd 3 is typical for this kind of shell script to open for its `/proc/self/fd` and there's only a very small probability that it will open another fd for it.